src/Microsoft.OData.Client/Serialization/HttpWebRequestMessage.cs

```diff
@@ -13,40 +13,14 @@ namespace Microsoft.OData.Client
     using System.IO;
     using System.Globalization;
     using System.Net;
-#if PORTABLELIB
-    using System.Reflection;
-    using System.Xml;
-#endif
     using Microsoft.OData;
 
     /// <summary> IODataRequestMessage interface implementation. </summary>
     [SuppressMessage("Microsoft.Design", "CA1001:TypesThatOwnDisposableFieldsShouldBeDisposable", Justification = "Returning MemoryStream which doesn't require disposal")]
+    [Obsolete("Migrate from the use of HttpWebRequest to HttpClient")]

```

Comment:
I think we should tell them to migrate from `HttpWebRequestMessage` to `HttpClientRequestMessage`.